bin/dasht-query-line

```diff
@@ -42,26 +42,24 @@
 #
 # ### Results
 #
-# Each search result is printed to stdout as a group of four lines of text:
+# Each search result is printed to stdout as a tab-separated line with fields:
```

Comment:
The fields are separated, not the line; so "as a line with 4 tab-separated fields".